packages/terra-action-footer/src/_ActionFooterContainer.jsx

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames/bind';
+import 'terra-base/lib/baseStyles';
+import styles from './ActionFooterContainer.scss';
+
+const cx = classNames.bind(styles);
+
+const propTypes = {
+  children: PropTypes.node,
+};
+
+const defaultProps = {
+  children: [],
+};
+
+const ActionFooterContainer = ({
+  children,
+  ...customProps
+}) => {
+  const isEmpty = !React.Children.count(children);
+
+  const actionFooterContainerClassNames = cx([
+    'action-footer-container',
+    { 'is-empty': isEmpty },
+    { 'has-children': !isEmpty },
+    customProps.className,
+  ]);
+
+  return (isEmpty ?
+    <div
```

Comment:
nit: these could probably be oneliners

Reply:
Yeah.. originally with the spacer the `{...padding} className={actionFooterContainerClassNames}` pushed it past 80cols

The first one for sure can be one line, but the second div with children pushes it to 87 cols.. Could always just rename `actionFooterContainerClassNames` to something shorter.